app/src/main/java/by/lomazki/pokemon/constants/Constants.kt

```diff
@@ -0,0 +1,9 @@
+package by.lomazki.pokemon.constants
+
+class Constants {
```

Comment:
класс с статическими полями это object
```suggestion
object Constants {
```